lib/ember-cli/app.rb

```diff
@@ -16,6 +16,10 @@ def compile
       silence_stream(STDOUT){ exec command }
     end
 
+    def install_dependencies
+      exec "npm install"
```

Comment:
How about we use our `which` helper to locate the `npm` bin here? We could cache it in configuration as well.

Something like this:

```ruby
class EmberCLI::Configuration
  def npm_path
    @npm_path ||= which("npm")
  end
end

class EmberCLI::App
  delegate :npm_path, to: :configuration
  def install_dependencies
    exec "#{npm_path} install"
  end
end